openfisca_us/variables/demographic/person.py

```diff
@@ -43,3 +43,57 @@ def formula(person, period, parameters):
             [age < 18, age < 65, age >= 65],
             [AgeGroup.CHILD, AgeGroup.WORKING_AGE, AgeGroup.SENIOR],
         )
+
+
+class ccdf_age(Variable):
+    value_type = float
+    entity = Person
+    label = u"CCDF age"
+    definition_period = YEAR
+
+
+class CCDFCareLocation(Enum):
+    CENTER_BASED = "Center-based"
+    HOME_BASED = "Home-based"
+
+
+class ccdf_care_location(Variable):
+    value_type = Enum
+    possible_values = CCDFCareLocation
+    default_value = CCDFCareLocation.CENTER_BASED
+    entity = Person
+    label = u"CCDF care location"
+    definition_period = YEAR
+
+
+class CCDFAgeGroup(Enum):
+    I = "Infant"
+    T = "Toddler"
+    PS = "Preschooler"
+    SA = "School age"
+
+
+class ccdf_age_group(Variable):
+    value_type = Enum
+    possible_values = CCDFAgeGroup
+    default_value = CCDFAgeGroup.I
+    entity = Person
+    label = u"CCDF age group"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        ccdf_age = person("ccdf_age", period)
+        ccdf_care_location = person(
+            "ccdf_care_location", period
+        ).decode_to_str()
```

Comment:
Ideally we'd want to compare Enums instead of comparing strings (it's slightly more efficient). For example:
```python
ccdf_care_location = person("ccdf_care_location", period)
care_locations = ccdf_care_location.possible_values # links to the Enum class
...
return select(
    [
        ((ccdf_age < 1.5) & (ccdf_care_location == care_locations.CENTER_BASED))
...
```